buildlib/jucx/jucx-test.yml

```diff
@@ -0,0 +1,63 @@
+jobs:
+  - job: jucx_test
+    displayName: Run jucx_test
+
+    pool:
+      name: MLNX
+
+    strategy:
+      matrix:
+        java8:
+          JAVA_VERSION: 1.8
+        java11:
+          JAVA_VERSION: 1.11
+
+    steps:
+      - checkout: self
+        clean: true
+        displayName: Checkout
+      - bash: |
+          set -eEx
+          ./autogen.sh
+          ./contrib/configure-release --prefix=$(Build.Repository.LocalPath)/install --with-java
+          make -j`nproc`
+          make install
+      - bash: |
+          source buildlib/az-helpers.sh
+          module load dev/mvn dev/jdk-${JAVA_VERSION}
+          make -C $(Build.Repository.LocalPath)/install/bindings/java/src/main/native test
+          ifaces=`ibdev2netdev | grep Up | awk '{print $5}'`
+          if [ -n "$ifaces" ]
```

Comment:
why this condition?

Reply:
To make sure there's RDMA interface on instance.

Reply:
but `make -C $(Build.Repository.LocalPath)/install/bindings/java/src/main/native package` only builds, not runs tests
why not build anyway